source/backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -97,33 +88,44 @@ private static ExpressionStarter<PimsProperty> GenerateCommonPropertyQuery(Claim
 
             var isRetired = filter.Ownership.Contains("isRetired");
 
-            ExpressionStarter<PimsProperty> ownershipBuilder;
+            ExpressionStarter<PimsPropertyLocationVw> ownershipBuilder;
 
             if (filter.Ownership.Count > 0)
             {
-                ownershipBuilder = isRetired ? PredicateBuilder.New<PimsProperty>(p => p.IsRetired == true) : PredicateBuilder.New<PimsProperty>(p => false);
+                var authorizationTypes = new List<string>() {
+                   "NOI",
+                   "Section 15",
+                   "Section 16",
+                   "Section 17",
+                   "Section 66",
+                };
+
+                // Property ownership filters
+                // TODO: Instead of doing redoing the query, the view could be used for ownership related queries
+                ownershipBuilder = isRetired ? PredicateBuilder.New<PimsPropertyLocationVw>(p => p.IsRetired == true) : PredicateBuilder.New<PimsPropertyLocationVw>(p => false);
                 if (filter.Ownership.Contains("isCoreInventory"))
                 {
                     ownershipBuilder = ownershipBuilder.Or(p => p.IsOwned && p.IsRetired != true);
                 }
-                /* TODO: Fix mapings
                 if (filter.Ownership.Contains("isPropertyOfInterest"))
                 {
-                    ownershipBuilder = ownershipBuilder.Or(p => p.IsPropertyOfInterest && p.IsRetired != true);
+                    ownershipBuilder.Or(p => p.IsRetired != true && p.HasActiveAcquisitionFile.HasValue && p.HasActiveAcquisitionFile.Value);
+                    ownershipBuilder.Or(p => p.IsRetired != true && p.HasActiveResearchFile.HasValue && p.HasActiveResearchFile.Value);
                 }
                 if (filter.Ownership.Contains("isOtherInterest"))
                 {
-                    ownershipBuilder = ownershipBuilder.Or(p => p.IsOtherInterest && p.IsRetired != true);
+                    var today = DateOnly.FromDateTime(DateTime.Now);
```

Comment:
this looks like an unused variable. What was this supposed to be used for?

Reply:
good catch. I was using it before when doing the queries manually instead of using the view. Will remove.